src/vs/workbench/contrib/terminalContrib/stickyScroll/browser/terminalStickyScrollContribution.ts

```diff
@@ -60,6 +60,14 @@ export class TerminalStickyScrollContribution extends Disposable implements ITer
 		this._refreshState();
 	}
 
+	hideLock() {
+		this._overlay.value?.lockHide();
+	}
+
+	hideUnlock() {
+		this._overlay.value?.unlockHide();
+	}
```

Comment:
this terminology is a bit confusing ... could it just be `lock` and `unlock`? vs `hideLock` and `hideUnlock`

Reply:
Lock and unlock by themselves seem like they'd be related to positioning. Also it has another hide mechanism that this overrides